servers/src/main/java/tachyon/master/file/journal/SetTTLEntry.java

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
Similar to make comment about RPC, would it make sense to have a SetStateEntry instead? For that to work, we would need to figure out how to represent unset values of TTL / pinned / ...